js/main.js

```diff
@@ -1346,14 +1346,13 @@ function BlackboxLogViewer() {
 
             userSettingsDialog = new UserSettingsDialog($("#dlgUserSettings"), 
             function(defaultSettings) { // onLoad
-    
-            prefs.get('userSettings', function(item) {
-                if(item) {
-                            userSettings = item;
-                         } else {
-                             userSettings = defaultSettings;
-                         }
-                });                
+                prefs.get('userSettings', function(item) {
+                    if(item) {
```

Comment:
Missing space. :space_invader:

Reply:
Do you mean there should be a space between if and (item) ? Otherwise if think the indents are more correct than before since it's in the onLoad function scope.

Reply:
Yes, `if (item) {` please.

Reply:
```suggestion
                    if (item) {
```